11-payment-encoding.md

```diff
@@ -190,7 +190,7 @@ A writer:
   - if `9` contains non-zero bits:
     - SHOULD use the minimum `data_length` possible.
   - otherwise:
-    - MUST omit the `9` field altogether.
+    - MUST omit the `9` and `s` field altogether and follow the old BOLT11 protocol.
```

Comment:
I don't think it's a good idea to mention an "old BOLT11 protocol".
The spec should only reflect the latest protocol (which is backwards-compatible).
I would remove this.